File: parantier-api/src/main/java/com/mapo/palantier/issue/domain/IssueMindmap.java
package com.mapo.palantier.issue.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Builder;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class IssueMindmap {
    private Long id;
    private Long issueId;
    private String title;
    private String content;
    private Integer orderNum;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: parantier-front/src/entities/issue/types/issueMindmap.ts
export interface IssueMindmap {
  id: number
  issueId: number
  title: string
  content: string
  orderNum: number
  createdAt: string
  updatedAt: string
}

export interface CreateMindmapRequest {
  title: string
  content: string
  orderNum?: number
}

export interface UpdateMindmapRequest {
  title: string
  content: string
  orderNum: number
}
